mautic/mautic
# More tooltips

[//]: # ( Please answer the following questions: )

| Q  | A
| --- | ---
| Bug fix? | 
| New feature? | y
| Related user documentation PR URL | 
| Related developer documentation PR URL | 
| Issues addressed (#s or URLs) | 
| BC breaks? | 
| Deprecations? | 

[//]: # ( Note that all new features should have a related user and/or developer documentation PR in their respective repositories. )

[//]: # ( Required: )
#### Description: 
Two tooltips added - one to the checkall button on the lead list view, and one to the filters tab in the edit segment view

#### Steps to test this PR:
1. Dashboard > Contacts
2. Hover over checkall button in top left of list table (check button with inactive dropdown next to it)
3. Tooltip should appear
4. Dashboard > Segments
5. Select a segment > Edit Segment
6. Hover over "filters" tab
7. Tooltip should appear
